pkg/templates/kubeadm/kubeadm.go

```diff
@@ -28,6 +28,7 @@ var (
 	v13x = mustParseConstraint("1.13.x")
 	v14x = mustParseConstraint("1.14.x")
 	v15x = mustParseConstraint("1.15.x")
+	v16x = mustParseConstraint("1.16.x")
```

Comment:
Can we simplify this with something like `>= 1.15.x`?

Reply:
It's done this way to be future-proof. `>= 1.15.x` is moving target (will be valid for anything after 1.15), while  `1.16.x` is constrained.